includes/global-functions.php

```diff
@@ -203,3 +240,348 @@ function distributor_get_registered_data() {
 
 	return $distributor_registered_data;
 }
+
+/**
+ * Pre-distribute callback for media data.
+ * This is the default pre-distribute callback for media data, used when the type is set to 'media' in distributor_register_data().
+ *
+ * @param mixed $media_id The media ID to be processed before distribution.
```

Comment:
Technically this could be called with a `WP_Post` object due to the way `get_post()` works but I think we can document it as an integer.

```suggestion
 * @param int $media_id The media ID to be processed before distribution.
```

Reply:
Handled in https://github.com/10up/distributor/pull/1309/commits/2c6994ec7ed58c064d282cc2929f128b2b33a428